flannel/src/pages/Profile.js

```diff
@@ -0,0 +1,266 @@
+import React, {useState,useEffect} from "react";
+import {useNavigate} from 'react-router-dom';
+import {Button, FormControl, InputLabel,Input, Select, MenuItem, TextField,Box,Typography} from '@mui/material'
+import InstagramIcon from '@mui/icons-material/Instagram';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import ChipFilter from '../components/ChipFilter'
+import { useCookies } from 'react-cookie';
+import logo from '../assets/bearLogo.png'
+
+const styles = {
+    root: {
+        display:"flex",
+        flexDirection: "column",
+        alignItems:"center",
+        padding: 3
+    },
+    rowContainer:{
+        display:"flex",
+        flexDirection:"row",
+        width:"95%"
+    },
+    title: {
+        paddingRight: 2,
+        fontFamily: 'Work Sans',
+        fontSize: 40,
+        marginBottom: 3,
+        fontWeight: 'bold',
+    },
+    logoContainer: {
+        padding: 2,
+        display: 'flex',
+        flexDirection: 'row',
+        width: 300,
+    },
+    formContainer:{
+        padding:5,
+        display:"flex",
+        flexDirection:"row",
+        boxShadow: '0 0px 4px rgba(0, 0, 0, 0.3)',
+        borderRadius: 4,
+    },
+    formColumn:{
+        display:"flex",
+        flexDirection:"column",
+        justifyContent:"space-around"
+    },
+}
+
+export default function  Profile(){
+    const user = JSON.parse(localStorage.getItem('user'));
+    console.log(user);
+    const [schoolYear,setSchoolYear] = useState(user.year);
+    const [password,setPassword] = useState("");
+    const [confirmPassword,setConfirmPassword] = useState("");
+    const [pronouns,setPronouns] = useState(user.pronouns);
+    const [signupError,setSignupError] = useState(false);
+
+    //selected tags
+    const [selectedClassTags,setSelectedClassTags] = useState([]);
+    const [selectedAffiliationTags,setSelectedAffiliationTags] = useState([]);
+    const [selectedInterestTags,setSelectedInterestTags] = useState([]);
+
+    //available tag options
+    const [classesTagOptions, setClassesTagOptions] = useState([])
+    const [interestsTagOptions, setInterestsTagOptions] = useState([])
+    const [affiliationsTagOptions, setAffiliationsTagOptions] = useState([])
+
+    const navigate = useNavigate();
+    useEffect(() => {
+        setClassesTagOptions(['CS 31', 'MATH 32A', 'PHYSICS 1A', 'BIO 1'])
+        setInterestsTagOptions(['Biking', 'Skating', 'Dancing'])
+        setAffiliationsTagOptions(['Theta Chi', 'DevX', 'GlobeMed', 'Climbing Club'])
+    }, []);
```

Comment:
this is good for now, but we'll just have to update this to use getLabels endpoint or the users own interests in the future